languagetool-language-modules/no/src/main/java/org/languagetool/rules/no/MorfologikNorwegianSpellerRule.java

```diff
@@ -0,0 +1,47 @@
+package org.languagetool.rules.no;
+
+import org.languagetool.Language;
+import org.languagetool.UserConfig;
+import org.languagetool.rules.spelling.morfologik.MorfologikSpellerRule;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public final class MorfologikNorwegianSpellerRule extends MorfologikSpellerRule {
```

Comment:
would be good to double check if Swedish (`sv`) and Danish (`da`) work (these 2 are used by our spelling checker https://github.com/Adthena/adthena-adwords/blob/dev/src/main/scala/adthena/detectors/LanguageToolDetectorService.scala) and why these 2 don't need `MorfologikSpellerRule`?

Reply:
they are using hunspell rules